app/assets/stylesheets/components/indicators.scss

```diff
@@ -26,23 +28,33 @@
 .crayons-indicator {
   font-family: $ff-accent;
   font-size: $fs-xs;
-  padding: $su-1;
-  background-color: var(--button-secondary-bg);
+  padding: 0.3335em 0.25em;
```

Comment:
@ludwiczakpawel How come we are moving away from the `$su-1` padding? I though that made it look pretty balanced.

This proposal to have the top and bottom padding larger than the horizontal accentuates making the indicator look skinny and squashed, especially with the skinnier typeface.

Reply:
Good question. By using EMs I wanted to make them a little bit more flexible but it seems like it's not really necessary so I've just brought back `$su`